Index: index.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\r\n<html lang=\"en\">\r\n<head>\r\n    <meta charset=\"UTF-8\">\r\n    <title>Title</title>\r\n</head>\r\n<body>\r\n\r\n</body>\r\n</html>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/index.html b/index.html
--- a/index.html	(revision 9bab62ee41c47667eca723cb2a2567227ef019f5)
+++ b/index.html	(date 1707495539117)
@@ -3,8 +3,11 @@
 <head>
     <meta charset="UTF-8">
     <title>Title</title>
+	<link rel="stylesheet" href="style.css">
 </head>
 <body>
 
+
+<script src="/main.js"></script>
 </body>
 </html>
\ No newline at end of file
Index: main.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.js b/main.js
new file mode 100644
--- /dev/null	(date 1707514860760)
+++ b/main.js	(date 1707514860760)
@@ -0,0 +1,44 @@
+// 3.Написати функцію заповнення даними користувача двомірного масиву.
+// Довжину основного масиву і внутрішніх масивів задає користувач.
+// Значення всіх елементів всіх масивів задає користувач.
+
+// const mainN = prompt('Введите длину главного массива');
+// const subN = prompt('Введите длину внутренних массивов');
+// const arr = new Array(+mainN).fill(0)
+//     .map((_, ind) =>
+//         new Array(+subN).fill(0).map((_, i) => prompt(`Введите элемент ${i} в подмассиве ${ind}`)));
+// alert('Ваш массив:' + arr.map(subArr => '\n' + subArr));
+
+// function doarr(mainN, subN) {
+//     const arr = new Array(+mainN).fill(0)
+//     .map((_, ind) =>
+//         new Array(+subN).fill(0).map((_, i) => prompt(`Введите элемент ${i} в подмассиве ${ind}`)));
+//     console.log('Ваш массив:' + arr.map(subArr => '\n' + subArr));
+//
+// }
+// doarr(3, 4);
+
+//1.Дано масив з елементами різних типів.
+// Створити функцію яка вираховує середнє арифметичне лише числових елементів даного масиву.
+
+let arr = [1, "abc", 2, 3, 4];
+let arrN = [];
+
+function creatArr(array) {
+    for(let i = 0; i < array.length; i++) {
+        if(typeof(array[i]) === "number") {
+
+            arrN.push(array[i]);
+        }
+
+
+    }
+
+    let sum = arrN.reduce(function (item) {
+        return sum += arrN[i];
+
+    }, 0);
+
+}
+
+console.log(creatArr(arr));
\ No newline at end of file
Index: style.css
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/style.css b/style.css
new file mode 100644
--- /dev/null	(date 1707426756669)
+++ b/style.css	(date 1707426756669)
@@ -0,0 +1,3 @@
+#text {
+    cursor: pointer;
+}
\ No newline at end of file
